### PR TITLE
(FM-6813) fix parsing for test tiers

### PR DESCRIPTION
**Pull request by ThoughtCrhyme**

### CODE REVIEW

File: lib/puppetlabs_spec_helper/rake_tasks.rb [hunk 1]
```diff
@@ -61,16 +61,14 @@
   t.pattern = 'spec/acceptance'
   # TEST_TIERS env variable is a comma separated list of tiers to run. e.g. low, medium, high
   if ENV['TEST_TIERS']
-    tiers = '--tag '
-    test_tiers = ENV['TEST_TIERS'].split(',')
+    test_tiers = test_tiers.split(',')
```
Comment (glennsarti) [inline lib/puppetlabs_spec_helper/rake_tasks.rb]: This is splitting on itself.  Shouldn't this still be `test_tiers = ENV['TEST_TIERS'].split(',')` ?

**Comment by glennsarti** [comment]
Nit: Commit message could be better.

**Comment by DavidS** [comment]
What @glennsarti said.

The amount of back and forth over this makes me wonder if there is a way to (unit) test this code on travis.

File: lib/puppetlabs_spec_helper/rake_tasks.rb (hunk 1 above)
Comment (ThoughtCrhyme) [reply lib/puppetlabs_spec_helper/rake_tasks.rb]: Yup, I removed a gsub call from earlier that botched this up. Thanks, Glenn

**Comment by ThoughtCrhyme** [comment]
@DavidS agreed, I'll look into it

**Pull request closed without merging**